integration-tests/bats/sql-server.bats

```diff
@@ -1542,6 +1542,30 @@ databases:
     [[ "$output" =~ "database is locked to writes" ]] || false
 }
 
+@test "sql-server: start server without socket flag should set default socket path" {
+    skiponwindows "unix socket is not available on Windows"
+    cd repo2
+    DEFAULT_DB="repo2"
+    let PORT="$$ % (65536-1024) + 1024"
```

Comment:
(future-refactoring) Not necessary for this PR, but just an idea for future refactoring/cleanup... it would be cool if we had a standard helper function we could call that would select a good port and potentially even check that it's not in use yet. Looks like this port selection code is in this BATS file a couple dozen times, so good candidate for a helper function. Not needed for this review, just sharing a future refactoring thought.

Reply:
Speaking of... this actually bit me this afternoon in a BATS run! one of the `sql-server` tests selected a port that was already in use and I had to restart the run. This would be a nice little cleanup task for the sql-server tests. 😄